style.css

```diff
@@ -256,51 +251,43 @@ a {
 }
 
 .contact {
-  width: 375px;
   border: 2px solid #fff;
   border-top-left-radius: 75px;
   color: white;
   background-color: #6070ff;
-  background-image: url("./img/contact_form_mobile.png");
+  background-image: url("./img/contact_form_desktop.png");
   background-repeat: no-repeat;
-  background-size: cover;
   display: flex;
   flex-direction: column;
-  justify-content: center;
   left: 328px;
+  width: 100%;
+  grid-column: 1/-1;
+  justify-content: center;
+  background-position: left center;
+  background-size: contain;
 }
 
```

Comment:
- [ ] Kindly fix here your margins on this size, since this should be responsive for the desktop version

![image](https://user-images.githubusercontent.com/93630700/214734894-3b7e89e1-8e1e-41b5-85fc-0c401ac14423.png)